docs/api/IoT.js-API-File-System.md

```diff
@@ -112,6 +112,161 @@ fs.closeSync(fd);
 ```
 
 
+## Class: fs.ReadStream
+
+A successful call to `fs.createReadStream()` will return a new `fs.ReadStream`
+object.
+
+`fs.ReadStream` inherits from `stream.Readable`.
+
+
+### Event: 'open'
+* `fd` {integer} File descriptor used by the `fs.ReadStream`.
+
+Emitted when the `fs.ReadStream`'s file descriptor has been opened.
+
+
+### Event: 'ready'
+
+Emitted when the `fs.ReadStream` is ready to be used. Emitted immediately
+after `open`.
+
+
+### Event: 'close'
+
+Emitted when the `fs.ReadStream`'s file descriptor has been closed.
+
+
+### ReadStream.bytesRead
+* {integer}
+
+Number of bytes that have been read so far.
+
+
+### ReadStream.path
+* {string}
+
+The path to the file of the `fs.ReadStream`.
+
+
+### fs.createReadStream(path[, options])
+* `path` {string} File path to open for reading.
+* `options` {Object}
+  * `flags` {string} Flags to open file with. **Default:** `'r'`
+  * `encoding` {string} **Default:** `null`
+  * `fd` {integer} File descriptor to be used. **Default:** `null`
+  * `mode` {integer} Permission mode. **Default:** `0666`
+  * `autoClose` {boolean} Should the file be closed automatically. **Default:** `true`
+  * `bufferSize` {integer} Size of buffer in bytes. **Default:** `4096`
+* Returns: `fs.ReadStream`
+
+If `fd` is specified, `path` will be ignored and the specified file
+descriptor will be used instead.
+
+If `autoClose` is `false`, the file will not be closed automatically,
+even if there is an error, it will be the application's responsibility.
+If it is `true` (as by default), the file will be closed automatically
+when end of file is reached or the stream ends.
+
+**Example**
+
+```js
+var fs = require('fs');
+
+var rStream = fs.createReadStream('example.txt');
+rStream.on('data', function(data) {
```

Comment:
The example uses the `data` event, but it is not documented at all here. However there are other events which are documented. It would be good to have a bit of info about the `data` also (if I remember correctly this is inherited from the readeble stream).